src/api/app/controllers/webui/request_controller.rb

```diff
@@ -272,8 +272,8 @@ def set_bugowner_request
 
         opts = { target_project: params[:project] }
         opts[:target_package] = params[:package] if params[:package]
-        opts[:person_name] = params[:user] if params[:user]
-        opts[:group_name] = params[:group] if params[:group]
+        opts[:person_name] = params[:user] if params[:user].present?
```

Comment:
why is important is `person_name` is nil or empty String? Does it make a difference? :thinking: